spec/install/path_spec.rb

```diff
@@ -49,6 +49,37 @@
       expect(vendored_gems("gems/rack-1.0.0")).to be_directory
       expect(the_bundle).to include_gems "rack 1.0.0"
     end
+
+    context "with path_relative_to_cwd set to true" do
+      before { bundle! "config path_relative_to_cwd true" }
+
+      it "installs the bundle relatively to current working directory", :bundler => "< 2" do
```

Comment:
why are we setting `:bundler => "< 2"` here?

Reply:
because this test expects the gem to be _available_, which it won't be on 2.0